docs/SEMVER_example

```diff
@@ -0,0 +1,3 @@
+1.0.0
+29/08/2023 08:05:21
```

Comment:
Why use custom formatted datetime string and implement custom parser for it? I think there should be enough of RFC3339 formated dt string.

Reply:
fixed